### ISSUE TITLE
error when generating dev.docker-compose.yml file

### ISSUE THREAD

**Issue by rvalyi** [akretion/voodoo#59]
with voodoo commit 89f1ae9a17eaeaa98f0d95b9550efaaf716ccb1d

```
rvalyi@rvalyi-laptop:~/DEV/heroku-instant-deploy$ voodoo run
The Voodoo Configuration have been updated, please take a look to the new config file
Update default config file at /home/rvalyi/.voodoo/config.yml
There is not dev.docker-compose.yml file.
Do you want to generate one automatically? [Y/n] y
Traceback (most recent call last):
  File "/usr/local/bin/voodoo", line 9, in <module>
    load_entry_point('voodoo-cli==2.1.1', 'console_scripts', 'voodoo')()
  File "/usr/local/lib/python2.7/dist-packages/voodoo/main.py", line 284, in main
    Voodoo.run()
  File "/usr/local/lib/python2.7/dist-packages/plumbum/cli/application.py", line 485, in run
    inst, retcode = subapp.run(argv, exit = False)
  File "/usr/local/lib/python2.7/dist-packages/plumbum/cli/application.py", line 457, in run
    inst = cls(argv.pop(0))
  File "/usr/local/lib/python2.7/dist-packages/voodoo/main.py", line 165, in __init__
    self.run_hook(GenerateDevComposeFile)
  File "/usr/local/lib/python2.7/dist-packages/voodoo/main.py", line 140, in run_hook
    return subcls(self, logger).run()
  File "/usr/local/lib/python2.7/dist-packages/voodoo/hook.py", line 96, in __init__
    template = pkg_resources.resource_stream(__name__, resource_path)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1178, in resource_stream
    self, resource_name
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1577, in get_resource_stream
    return open(self._fn(self.module_path, resource_name), 'rb')
IOError: [Errno 2] No such file or directory: '/usr/local/lib/python2.7/dist-packages/voodoo/locomotive/docker-compose.yml'
```

**Comment by sebastienbeau** [comment]
hum I got it! let's fix it

**Comment by sebastienbeau** [comment]
fixed in commit 19901a09dcbcbbac52547d9e4c26e6f5c4a5bd0a
